doc/crypto/api/ops/kdf.rst

```diff
@@ -906,18 +910,18 @@ Key derivation functions
     .. param:: const uint8_t *expected_output
         Buffer containing the expected derivation output.
     .. param:: size_t output_length
-        Length ot the expected output. This is also the number of bytes that will be read.
+        Length of the expected output. This is also the number of bytes that will be read.
 
     .. return:: psa_status_t
     .. retval:: PSA_SUCCESS
         Success.
         The output of the key derivation operation matches ``expected_output``.
     .. retval:: PSA_ERROR_NOT_PERMITTED
-        One of the inputs is a key whose policy does not permit `PSA_KEY_USAGE_VERIFY_DERIVATION`.
+        One of the inputs is a key whose policy permits neither `PSA_KEY_USAGE_DERIVE` nor `PSA_KEY_USAGE_VERIFY_DERIVATION`.
```

Comment:
If I understand this correctly, the key given as input to `psa_key_derivation_input_key` must have policy as `PSA_KEY_USAGE_DERIVE` or `PSA_KEY_USAGE_VERIFY_DERIVATION`. Is this correct?

Reply:
Correct.